src/main/scala/com/fulcrumgenomics/util/PickIlluminaIndices.scala

```diff
@@ -0,0 +1,59 @@
+package com.fulcrumgenomics.util
+
+import scala.collection.JavaConversions._
+
+import com.fulcrumgenomics.cmdline.{ClpGroups, JeanLucTool}
+import dagr.commons.CommonsDef.{DirPath, FilePath}
+import dagr.sopt.{arg, clp}
+import picard.util.IlluminaUtil.IlluminaAdapterPair
+
+/**
+  * Program for picking sets of indices of arbitrary length that meet certain constraints
+  * and attempt to maximize the edit distance between all members of the set picked.
+  *
+  * @author Tim Fennell
+  */
+@clp(
+ description = "Picks a set of molecular indices that should work well together.",
+ group = classOf[ClpGroups.Utilities])
```

Comment:
I was wondering if we should have a typedef `type UtilitiesGroup = classOf[ClpGroups.Utilities]` to make it clearer than `classOf[...]`?